src/compiler/crystal/codegen/fun.cr

```diff
@@ -382,7 +395,14 @@ class Crystal::CodeGenVisitor
   def setup_context_fun(mangled_name, target_def, llvm_args_types, llvm_return_type) : Nil
     context.fun = @llvm_mod.functions.add(mangled_name, llvm_args_types, llvm_return_type, target_def.varargs?)
 
-    context.fun.add_attribute LLVM::Attribute::AlwaysInline if target_def.always_inline?
+    unless @debug.variables?
```

Comment:
Can we use `if @debug.variables?` then swap the order of these branches?

Reply:
No problem. Done.

Reply:
Doesn't look like this has been pushed.

Reply:
I am testing changes just in case. Will be pushed in a few minutes.